doc/source/Environments.rst

```diff
@@ -18,7 +18,7 @@ Environments can be constructed in two ways in spack-stack:
            spack add netcdf-c@4.7.4
            spack add ufs-weather-model-env
            spack add ufs-weather-model-env@1.0.0 +debug
-           spack add jedi-ewok-env +solo +r2d2 +ewok 
+           spack add ewok-env
```

Comment:
Why were the variants (solo, r2d2, ewok) removed? I didn't see a change in the companion spack PR that might have accounted for this change, but I realize that the variant specs here might have been redundant. Just want to double check.

Reply:
I decided not to remove the variants from the ewok-env package yet, just in case we want to use them again in the future. But since we haven't updated these packages in spack in a while, and since they are still not publicly available, I thought it would be best to not enable them here in this example.

Reply:
Got it - that makes sense. Thanks for the explanation.